apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.jaxrs;
+
+import org.stagemonitor.configuration.ConfigurationOption;
+import org.stagemonitor.configuration.ConfigurationOptionProvider;
+
+/**
+ * Configuration provider for the apm jax-rs plugin
+ */
+public class JaxRsConfiguration extends ConfigurationOptionProvider {
+    private static final String JAXRS_CATEGORY = "JAX-RS";
+
+    private final ConfigurationOption<Boolean> allowPathOnHierarchy = ConfigurationOption.booleanOption()
+        .key("allow_path_on_hierarchy")
```

Comment:
Just from looking at the configuration name it's not immediately apparent that it's about JAX-RS annotations.
I suggest renaming to `enable_jaxrs_annotation_inheritance`